core/arch/arm/kernel/spmc_sp_handler.c

```diff
@@ -1106,6 +1106,48 @@ static void spmc_handle_version(struct thread_smc_args *args,
 		      FFA_PARAM_MBZ, FFA_PARAM_MBZ);
 }
 
+static void handle_console_log(struct thread_smc_args *args)
+{
+	uint32_t ret_fid = FFA_ERROR;
+	uint32_t ret_val = FFA_INVALID_PARAMETERS;
+	size_t char_count = (args->a1 & FFA_CONSOLE_LOG_CHAR_COUNT_MASK);
+	const void *char_lists[FFA_CONSOLE_LOG_REG_COUNT] = {
```

Comment:
Wouldn't it make more sense to name this `reg_list`?
Also, `[FFA_CONSOLE_LOG_REG_COUNT]` isn't needed, `[]` should be enough.

Reply:
Ok, I've changed it to `reg_list`.